common/isolationgroup/default-isolation-group-state.go

```diff
@@ -56,8 +56,7 @@ func NewDefaultIsolationGroupStateWatcher(
 	logger log.Logger,
 	domainCache cache.DomainCache,
 	config Config,
-	globalIsolationGroupDrains persistence.ConfigStore,
-	done chan bool,
```

Comment:
this param is supposed to be removed?

Reply:
no, I didn't intend to put that change here. once again, thank you